string name
float64 xpos
float64 ypos
float64 distance
bool is_collected
geometry_msgs/Pose2D robot_pose
visualization_msgs/Marker marker

